api.js

```diff
@@ -175,6 +175,7 @@ class Api extends Emittery {
 						if (apiOptions.concurrency > 0) {
 							concurrency = apiOptions.concurrency;
 						}
+
```

Comment:
This is for the next XO version. See: https://github.com/xojs/eslint-config-xo/pull/52